utils/callApi.js

```diff
@@ -19,6 +19,7 @@ const callApi = async ({ method, endpoint, body }, inputs) => {
 
   if (response.status !== 200) {
     logWarning(`Please ensure that Github App is installed ${GITHUB_APP_URL}`)
+    return response
```

Comment:
why are you returning the response? are we using it?

Reply:
why would you return it?

Reply:
to make sure no actions continue if not 200

Reply:
sounds good